### ISSUE TITLE
Question on OpenVLA mini

### ISSUE THREAD

**Issue by lostgreen** [openvla/openvla#252]
Hi, how can i eval openvla-mini? I notice that only pretrain and train scripts are availabel.

**Comment by Akila-Ayanthi** [comment]
Hi did you figure out how to do openvla-mini evals?
I used the experiments/robot/libero/run_libero_eval.py with the openvla-mini checkpoint available and I'm getting <5% accuracy.

**Comment by XXXUAN0904** [comment]
> Hi did you figure out how to do openvla-mini evals? I used the experiments/robot/libero/run_libero_eval.py with the openvla-mini checkpoint available and I'm getting <5% accuracy.

How did you complete the evaluation? Why do I get an error loading the pre-training checkpoints in minivla's hugging face with the missing field ‘model_type’ in the ‘config.json’ file?

**Comment by lostgreen** [comment]
Sorry, we still did not figure out how to use this model, we use another vla instead_

**Comment by Akila-Ayanthi** [comment]
I used model_family=prismatic and in the get_prismatic_vla_action() used the line
action, ids = vla.predict_action(processed_images, task_label, unnorm_key=unnorm_key, max_new_tokens=vla.get_action_dim(unnorm_key), **kwargs).

This gave better results but still not the numbers reported. 
Without max_new_tokens=vla.get_action_dim(unnorm_key), it was way worse, <5%.

**Comment by ylhaichen** [comment]
look at Converting Prismatic Models to Hugging Face section in readme, there was a note says "Converting and deploying MiniVLA models and VQ / multi image is not supported yet!"

**Comment by Akila-Ayanthi** [comment]
@ylhaichen Yes, so can we evaluate without converting?

**Comment by ylhaichen** [comment]
> [@ylhaichen](https://github.com/ylhaichen) Yes, so can we evaluate without converting?

I don't think so. A conversion seems needed but the script wasn't provided.
